sendRequests.py

```diff
@@ -0,0 +1,63 @@
+from tornado import ioloop
+import time
+from Host.balance import landing
+import sys
+
+s = time.time()
+maxRequestCount = 1000
+
+i = 0
+c = 0
+
+# Suppresses the error from printing
+class DevNull:
+    i = 0
+    c = 0
+    def write(self, msg):
+        #pass
+        # This part of code isn't working, check it out.
+        for j in range(1, 20):
+            i += 1
+            c += 1
+            print(c)
+            handle_request()
+
+sys.stderr = DevNull()
+
+def handle_request():
+    
+    res = landing()
+    
+    global i
+    i -= 1
+
+    # Must be used when client count is increased.
+    #time.sleep(0.2)
+
+    if i == 0:
+        ioloop.IOLoop.instance().stop()
+
+#1000 client requests are processed in 4 seconds.
+for j in range(maxRequestCount):
+    i += 1
+    c += 1
+    print(c)
+    handle_request()
+try:
+    ioloop.IOLoop.instance().start()
+except ConnectionRefusedError:
+    #pass
+    print("Interrupt occured, taking break for 2 seconds.")
+    time.sleep(2)
```

Comment:
Is this multi-threaded?

Reply:
Yes, `tornado.ioloop` enables us to send more than single request concurrently.